nucypher/blockchain/eth/agents.py

```diff
@@ -92,16 +103,11 @@ def get_balance(self, address: str=None) -> int:
     def approve_transfer(self, amount: int, target_address: str, sender_address: str) -> str:
         """Approve the transfer of token from the sender address to the target address."""
 
-        txhash = self.contract.functions.approve(target_address, amount)\
-            .transact({'from': sender_address})#, 'gas': 40000})  # TODO: needed for use with geth.
-
+        txhash = self.contract.functions.approve(target_address, amount).transact({'from': sender_address})#, 'gas': 40000})  # TODO: needed for use with geth.
```

Comment:
What's the issue for this?

Reply:
https://github.com/nucypher/nucypher/issues/413